src/recipes.jl

```diff
@@ -937,6 +937,8 @@ end
     mat = g.args[1]
     if length(unique(mat[mat .!= 0])) < 2
         legend --> nothing
+        # revert the default gradient to have a darker (almost black) color in the default theme
+        seriescolor --> cgrad([cgrad()[0], cgrad()[1]])
```

Comment:
why not just make it black?

Reply:
I thought this might look strange with `theme(:dark)`. But we could default to the axes color if you prefer a real black in the default case.

Reply:
No, you're right 👍